src/Lykke.Service.Subscribers/Controllers/SubscribersController.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Common.Log;
+using JetBrains.Annotations;
+using Lykke.Service.Subscribers.AzureRepositories.Subscribers;
+using Lykke.Service.Subscribers.Core.Repositories;
+using Lykke.Service.Subscribers.Models.Subsribers;
+using Lykke.Service.Subscribers.Strings;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.SwaggerGen;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Lykke.Service.Subscribers.Controllers
+{
+    [Route("api/subscribers")]
+    public class SubscribersController : Controller
+    {
+        private readonly ISubscriberRepository _subscriberRepository;
+        private readonly ILog _log;
+
+        public SubscribersController([NotNull]ISubscriberRepository subscriberRepository, [NotNull] ILog log)
+        {
+            _subscriberRepository = subscriberRepository;
+            _log = log;
+        }
+
+        /// <summary>
+        /// Get a subscribers.
+        /// </summary>
+        [HttpGet]
+        [SwaggerOperation("GetSubsribers")]
+        [ProducesResponseType(typeof(IEnumerable<SubscriberResponseModel>), (int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
```

Comment:
This code do not return 404. It is a wrong documentation, or a lack in implementation.